packages/backend/src/config/config.testing.ts

```diff
@@ -16,5 +16,22 @@ export function getTestConfig(): Config {
     },
     port: 1337,
     databaseUrl: getEnv('TEST_DB_URL', __SKIP_DB_TESTS__),
+    jsonRpcUrl: getEnv('TEST_JSON_RPC_URL', 'http://localhost:8545'),
+    core: throwsOnUnexpectedAccess({
```

Comment:
I like this, but it should probably enforce that all the keys from `core` are present.

Reply:
My bad, it actually is typesafe